### DISCUSSION TITLE
Funding to fix the build issue, and make SuperPleccer alive and solid!

### DISCUSSION THREAD

**Discussion by rvmn** [rvmn/SuperPleccer#27] [Polls]
I started SuperPleccer as an amibitous project with the experience of coding functional code and found that the underlying source is using dependencies that need updating and I don't have/cannot make the time to get this done in a reasonable amount of time.

So, the question lying on the table, is whether you would be ok chipping in a small amount to fix the compilation (building the app) issue and make it a living updateable project. 

I am building on features in the main dev fork (Pleccer) that I can add but it makes no sense to do so untill the app can be build and be used. 

Thanks in advance, and tell me if you have ideas!

BTW, i have found someone reliable to fix the issue, but in total will cost 250EU, which is decent, but not in my budget.

## Please see the link posted below for the fundraising campaign!

Greets,
Roberto

**Comment by rvmn** [comment]
## Please visit https://mtyc.co/me5m0g to support SuperPleccer build for Windows and Mac.
Thank you!

**Comment by Jan-Snijder** [comment]
Offcourse, but only if its kept alive after that :)
